sync-core/src/main/java/com/cloudant/sync/datastore/SavedHttpAttachment.java

```diff
@@ -36,7 +36,6 @@
     private URI attachmentURI;
     private int size;
     private byte[] data;
-    private Encoding encoding;
```

Comment:
Didn't notice `encoding`, fair enough.

After #143 merges then `length` and `encoded_length` will be stored as `public final` values in the appropriate `Attachment` classes and `getLength()` will be dropped from the `Attachment` interface. So I guess I can revisit this class then.